flake.nix

```diff
@@ -173,7 +173,7 @@
             zlib.out
             fenix.packages.${system}.rust-analyzer
             just
-            pkgconfig
```

Comment:
Oof did pkgconfig not exist? What's the difference between the two :eyes:

Reply:
The name had just been replaced with pkg-config! But otherwise it's the same. We were getting build errors from it

Reply:
Good catch!